vagrant/ansible/playbook.yml

```diff
@@ -0,0 +1,156 @@
+    command: apt-get install -y software-properties-common
+    become: true
+    
+  - name: Install https driver
+    command: apt-get install apt-transport-https
+    become: true
+
+  - name: Upload create tables file
+    template:
+      src: "files/config/default.properties"
+      dest: "/var/local"
+    become: true
+
+  - name: install setfacl support
+    become: yes
+    apt: pkg=acl
+
+  # install postgresql 9.5
+  - name: Configure the PostgreSQL APT key
+    apt_key: url=https://www.postgresql.org/media/keys/ACCC4CF8.asc state=present
+
+  - name: Configure the PostgreSQL APT repositories
+    apt_repository: repo="deb http://apt.postgresql.org/pub/repos/apt/ {{ ansible_distribution_release}}-pgdg main" state=present
+
+  - name: Install PostgreSQL
+    apt: pkg={{ item }} state=present
+    with_items:
+      - postgresql-9.5
+      - postgresql-server-dev-9.5
+    become: true
+
+  - name: Create reporting databse
+    postgresql_db:
+      name: reporting
+    become: true
+    become_user: postgres
+
+  - postgresql_user:
+      db: reporting
+      name: reportingDB
+      password: reportingDB
+      priv: ALL
+    become: true
+    become_user: postgres
+
+  - name: Overwrite postgresql.conf
+    lineinfile: 
+      dest=/etc/postgresql/9.5/main/postgresql.conf
+      regexp="^listen_addresses"
+      line="listen_addresses = '*'" state=present
+    become: true
+
+  - name: Overwrite pg_hba.conf
+    copy:
+      dest: "/etc/postgresql/9.5/main/pg_hba.conf"
+      content: |
+        local all postgres  md5
+        local all all     md5
+        host  all all     0.0.0.0/0 md5
+    become: true
+    notify: restart postgres
+
+  - name: Upload create tables file
+    template:
+      src: "files/postgresql/createSequencesAndTablesProductionDB.sql"
+      dest: "/home"
+
+  - name: Create Tables
+    bocome: true
+    become_user: postgres
+    shell: psql reporting < /home/createSequencesAndTablesProductionDB.sql
+    ignore_errors: true
+    notify: restart postgres
+
+  - name: Delete create tables file
+    command: rm /home/createSequencesAndTablesProductionDB.sql
+
+  # install openjdk 8
+  - name: Install openJDK 8
+    command: sudo apt-get -y install default-jdk
+    become: true
+    become_user: root
+    
+  - shell: echo JAVA_HOME is $JAVA_HOME
+    environment:
+      JAVA_HOME: /usr/lib/jvm/java-8-openjdk-amd64
+
+  - file:
+      src: /usr/lib/jvm/java-8-openjdk-amd64
+      dest: /usr/lib/jvm/default-java
+      state: link
+
+  # install tomcat 8
+  - name: Install Tomcat 8
+    package: name=tomcat8 state=latest
+
+  - name: Stop Tomcat
+    command: systemctl stop tomcat8.service
+
+  - name: Install Tomcat 8 Admin
+    package: name=tomcat8-admin state=latest
+
+  - name: Overwrite Tomcat 8 Usersr
+    template:
+      src: "templates/tomcat-users.xml"
+      dest: "/etc/tomcat8/tomcat-users.xml"
+  - name: create reporting folder
+    command: mkdir /var/lib/tomcat8/webapps/qucosa-fcrepo-reportingdb
```

Comment:
This works, however, there are nicer approaches for installing a web application to tomcat.

This way is not the way it's done on the production machines.